### ISSUE TITLE
createlang cannot find plperl

### ISSUE THREAD

**Issue by mmahalwy** [phlipper/chef-postgresql#133]
Getting this error:
```
    STDERR: createlang: language installation failed: ERROR:  could not open extension control file "/usr/share/postgresql/9.4/extension/plperl.control": No such file or directory
```

For some reason the cookbook is not installing all the extensions necessary. 
My runlist:
```
"recipe[postgresql]",
    "recipe[postgresql::client]",
    "recipe[postgresql::contrib]",
    "recipe[postgresql::service]",
    "recipe[postgresql::libpq]",
    "recipe[postgresql::setup_users]",
    "recipe[postgresql::setup_languages]",
    "recipe[postgresql::setup_databases]",
```

**Comment by mmahalwy** [comment]
Possibly allowing for: `install postgresql-plperl`

**Comment by phlipper** [comment]
@mmahalwy thank you for the report. This is now resolved as of #137 and has been released with `0.16.1`.

**Comment by mmahalwy** [comment]
@phlipper you're a BO$$
